README.md

```diff
@@ -4,7 +4,48 @@ EVA Submission Command Line Interface for Validation
 
 ## Installation
 
-TBD
+There are currently three ways to install and run the tool: using conda, from source using Docker,
+and from source natively (i.e. managing dependencies on your own).
+
+### Conda
```

Comment:
This is a bit presumptive as it hasn't been submitted to the channel yet, I'm assuming it will be fine :crossed_fingers: